File: ui/src/models/Pinboard.ts
export type PinboardPost = {
    id: string;
    clubId: string;
    clubName: string;
    authorId: string;
    authorName: string;
    title: string;
    body: string;
    pinned: boolean;
    createdAt: string;
    updatedAt: string;
};

export type PinboardPostUpsert = {
    title: string;
    body: string;
    pinned: boolean;
};

File: api_go/models/Pinboard.go
package models

import "time"

type PinboardPost struct {
	ID         string
	ClubID     string
	ClubName   string
	AuthorID   string
	AuthorName string
	Title      string
	Body       string
	Pinned     bool
	CreatedAt  time.Time
	UpdatedAt  time.Time
}
